src/main/java/com/google/sps/data/CareerQuestionChoice.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+
+/** Class to store the reponse text of each Choice in the career quiz, and what path
```

Comment:
Is this formatting correct?

And does the linter check catch it? Looks like maybe it's not set up correctly?